lightning.profile

```diff
@@ -2,101 +2,5 @@
 
 /**
  * @file
- * Defines the Lightning Profile install screen by modifying the install form.
+ * The Lightning profile.
```

Comment:
Do we still need the .profile? Doesn't look like there's going to be anything in it.